drivers/at86rf2xx/at86rf2xx_getset.c

```diff
@@ -305,6 +306,23 @@ void at86rf2xx_set_csma_seed(at86rf2xx_t *dev, uint8_t entropy[2])
     at86rf2xx_reg_write(dev, AT86RF2XX_REG__CSMA_SEED_1, tmp);
 }
 
+int8_t at86rf2xx_get_cca_threshold(at86rf2xx_t *dev)
+{
+    int8_t tmp = at86rf2xx_reg_read(dev, AT86RF2XX_REG__CCA_THRES);
+    tmp &= AT86RF2XX_CCA_THRES_MASK__CCA_ED_THRES;
+    tmp <<= 1:
+    return (RSSI_BASE_VAL + tmp);
+}
+
+void at86rf2xx_set_cca_threshold(at86rf2xx_t *dev, int8_t value)
+{
```

Comment:
Er... I don't get your point : CCA threshold values are certainly supposed to be negative (see the definition in the PR intro comment above). Shouldn't we on the contrary ensure the value is negative?...